src/utility/src/math_utils.h

```diff
@@ -32,7 +32,7 @@ static inline uint8_t clz(uint32_t n)
 #if defined WIN32
     return (uint8_t)(_lzcnt_u32(n));
 #elif defined __linux__ && (__GNUC__ >= 14)
-    return (uint8_t)(__builtin_clzg(n, (8 * sizeof(n)));
+    return (uint8_t)(__builtin_clzg(n, (int) (8 * sizeof(n))));
```

Comment:
Likewise, probably better as `int32_t`

Reply:
Same as above, the 2nd argument is document as an int